spec/deaf_grandma_spec.rb

```diff
@@ -7,12 +7,15 @@
   it "says 'SPEAK UP SONNY!' when we speak regularly" do
     expect(script.speak("Hi Grandma")).to eq "SPEAK UP SONNY!"
   end
-  
+
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HI GRANDMA WHEN DID YOU LAST EAT NOODLES.")).to eq "NOT SINCE 1964!"
+
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    expect(script.speak("BYE")).to eq "NOT SINCE 1964!"
```

Comment:
We only want Grandma to respond if you have yelled 3 times at her.  Think about how you could write a test to make sure the first two times you say bye, she doesn't respond.

Reply:
Made fix to return empty string